app/components/List/tests/index.test.js

```diff
@@ -1,15 +1,22 @@
 import expect from 'expect';
-import { render } from 'enzyme';
+import { mount } from 'enzyme';
 import React from 'react';
 
 import List from '../index';
 import ListItem from 'components/ListItem';
 
 describe('<List />', () => {
   it('should render the component if no items are passed', () => {
-    const renderedComponent = render(
+    const renderedComponent = mount(
       <List component={ListItem} />
     );
+    expect(renderedComponent.find(List)).toExist();
+  });
+
+  it('should not render the items if no items are passed', () => {
+    const renderedComponent = mount(
```

Comment:
It would be a little better performance-wise if we could test this using shallowRender instead of mount.
I can't test this right now but I'm fairly certain that we could write this test effectively using shallowRender